yarn.lock

```diff
@@ -27417,7 +27400,14 @@ __metadata:
   languageName: node
   linkType: hard
 
-"multiformats@npm:^9.4.2, multiformats@npm:^9.5.2":
+"multiformats@npm:^13.1.0":
+  version: 13.2.2
+  resolution: "multiformats@npm:13.2.2"
+  checksum: 10/6e673320e9b06d5fdbbf2bde0d3132f13fac94fb40f36d646265b5c38eba4a28c40a2c76b4efa0c1a23517fe87320e540e9ef7f28d71c1cc3239c91bf6770ce6
+  languageName: node
+  linkType: hard
+
+"multiformats@npm:^9.4.2":
```

Comment:
Separate: We should try to remove the remaining use of legacy `multiformats@9.x` in a separate PR. That might claw back some of the bundle-increased incurred by including different two versions simultaneously, here.